packages/mui-material/package.json

```diff
@@ -104,6 +104,14 @@
   "engines": {
     "node": ">=14.0.0"
   },
+  "exports": {
+    "/ButtonBase/TouchRipple": {
```

Comment:
Maintain backwards compatibilty for https://github.com/mui/mui-x/blob/1a9426021bd72b5a1cc18e2c78a5a063d27be541/docs/data/data-grid/column-definition/RenderCellGrid.tsx#L4